src/Module/module.css

```diff
@@ -91,88 +32,35 @@ $moduleSpacing:  3rem;
         padding-bottom:     3rem;
         margin-bottom:      0;
```

Comment:
This `padding-bottom` and `margin-bottom`, and the `padding-bottom` above on line 28 are all redundant.